packages/core/src/logs/eventMapper.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import { InternalLog } from '../InternalLog';
+import { SdkVerbosity } from '../SdkVerbosity';
+import { DdSdk } from '../foundation';
+import type { Attributes } from '../sdk/AttributesSingleton/types';
+import type { UserInfo } from '../sdk/UserInfoSingleton/types';
+
+import type {
+    LogEvent,
+    LogEventMapper,
+    LogStatus,
+    NativeLog,
+    RawLog
+} from './types';
+import { deepClone } from './utils/deepClone';
+
+export const applyLogEventMapper = (
+    logEventMapper: LogEventMapper,
+    log: LogEvent
+): NativeLog | null => {
+    const originalLog = deepClone(log);
```

Comment:
Not sure at all about this but this weeks Euro Brown Bag made me think, could this lead to Prototype pollution?

Reply:
No, for prototype pollution to happen it needs:
- parsing in some form
- user input

There is no parsing here, so even if developers choose to put some user input inside the context of their log, there will be no prototype pollution.
Globally we don't have to worry too much about prototype pollution as it will most likely strike in the application code first, so it's more of the application's developers responsibility to prevent this.